README.md

```diff
@@ -3,7 +3,8 @@
 [![Release](https://img.shields.io/github/release/aws/aws-sdk-ios.svg)](../../releases)
 [![CocoaPods](https://img.shields.io/cocoapods/v/AWSCore.svg)](https://cocoapods.org/pods/AWSCore)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
-[![CircleCI](https://circleci.com/gh/aws-amplify/aws-sdk-ios.svg?style=svg)](https://circleci.com/gh/aws-amplify/aws-sdk-ios)
+[![Integration Test](https://github.com/aws-amplify/aws-sdk-ios/actions/workflows/integ-test.yml/badge.svg)](https://github.com/aws-amplify/aws-sdk-ios)
+[![Unit Test](https://github.com/aws-amplify/aws-sdk-ios/actions/workflows/unit-test.yml/badge.svg)](https://github.com/aws-amplify/aws-sdk-ios)
```

Comment:
🙈  sorry, i meant if we should have a new workflow that runs in `main`. 
These badges will show the result of the **latest** run, which could be in any development branch.

It's fine to keep them for now, but this can be an ops improvement to do eventually.

Reply:
Yeah. But we don't run integration test or unit test on `main` branch.. If we configure with `main`, the badges would be like this:

![](https://github.com/aws-amplify/aws-sdk-ios/actions/workflows/integ-test.yml/badge.svg?branch=main)
![](https://github.com/aws-amplify/aws-sdk-ios/actions/workflows/unit-test.yml/badge.svg?branch=main)